internal/gencli/root_file.go

```diff
@@ -72,11 +72,14 @@ func printVerboseInput(srv, mthd string, data interface{}) {
 func printMessage(data interface{}) {
 	var s string
 
-	if OutputJSON {
-		d, _ := json.MarshalIndent(data, "", "  ")
-		s = string(d)
-	} else if msg, ok := data.(proto.Message); ok {
-		s = msg.String()
+	if msg, ok := data.(proto.Message); ok {
+		if OutputJSON {
+			var b bytes.Buffer
+			marshaler.Marshal(&b, msg)
+			s = b.String()
+		} else {
```

Comment:
Can you make the `else` case the "default" and move it above the `if` statement?  I like to reduce indentation and avoid `else` statements when possible.

```go
s = msg.String()
if OutputJSON {
    var b bytes.Buffer
    marshaler.Marshal(&b, msg)
    s = b.String()
}
```

Reply:
Sounds good! Done.